src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Swift/SwiftTypes.cs

```diff
@@ -19,7 +20,8 @@ namespace System.Runtime.InteropServices.Swift
     /// </code>
     /// </para>
     /// </remarks>
-    [CLSCompliantAttribute(false)]
+    [CLSCompliant(false)]
+    [Intrinsic]
```

Comment:
Is there going to be accompanying [named-intrinsics](https://github.com/dotnet/runtime/blob/08d98197668e5d62e4338730a427ab6b69d49b7c/src/coreclr/jit/namedintrinsiclist.h#L12) in JIT for these, or does marking type as intrinsic improves interop in other ways?

Reply:
Marking the type as an intrinsic makes it faster to filter down searches for it in the JIT (checking "type is intrinsic" before name matching allows us to skip name matching 99% of the time).